src/brouwers/kitreviews/templates/kitreviews/find_kit.html

```diff
@@ -15,50 +15,74 @@
     </form>
 
     {% if form.data %}
-    <h3>{% blocktrans with num_results=kits.count %}Search results ({{ num_results }}){% endblocktrans %}</h3>
+        <h3>{% blocktrans with num_results=kits.count %}Search results ({{ num_results }}){% endblocktrans %}</h3>
 
-    <section class="kit__list">
-        <div class="row">
-        {% for kit in kits %}
-            <div class="col-xs-6 col-sm-4 col-md-3 col-xl-2">
-                <article class="kit__preview" data-id="{{ kit.id }}">
+        {% if not kits.exists %}
+
+            <h4>
+                {% blocktrans %}Oops, we don't have this kit in our database! Would you like to add
+                    it?{% endblocktrans %}
+            </h4>
+
+            <div class="center">
+                <input type="button" data-target="#add-kit-modal" value="{% trans 'Add kit' %}"
+                       class="btn bg-main-orange"/>
+            </div>
+
+            {% add_kit_modal form=add_form data=form.cleaned_data %}
+
+        {% endif %}
+
+        <section class="kit__list">
+            <div class="row">
+                {% for kit in kits %}
+                    <div class="col-xs-6 col-sm-4 col-md-3 col-xl-2">
+                        <article class="kit__preview" data-id="{{ kit.id }}">
                     <span class="h5 kit__preview__name" title="{{ kit.name }}">
                         <strong>{{ kit.name }}</strong>
                     </span>
```

Comment:
I think my editor was setup for Jinja2 instead of Django templates, I'll look into that